expat/doc/xmlwf.xml

```diff
@@ -379,11 +394,19 @@ supports both.
   <refsect1>
   <title>OUTPUT</title>
     <para>
-	If an input file is not well-formed,
+	If any input file is not well-formed,
 	<command>&dhpackage;</command> prints a single line describing
-	the problem to standard output.  If a file is well formed,
-	<command>&dhpackage;</command> outputs nothing.
-	</para>
+	the problem to standard output. If the <option>-k</option> option is
+	provided, <command>&dhpackage;</command> continues processing the
+	remaining input files, describing problems found in any of them.
+	If <option>-k</option> is not provided, <command>&dhpackage;</command>
+	exits after processing the first input file with a problem.
+	<option>-k</option> outputs nothing for files which are well-formed.
```

Comment:
That seems to hold true for the case without `-k` as well, so it seems a bit mis-leading.